pa11ycrawler/spiders/edx.py

```diff
@@ -215,13 +239,17 @@ def after_initial_login(self, response):
         It verifies that the login request was successful,
         and then generates requests from `self.start_urls`.
         """
-        if "We couldn't sign you in." in response.text:
+        if LOGIN_FAILURE_MSG in response.text:
             self.logger.error("Credentials failed!")
```

Comment:
I wonder if you could make the error more helpful (e.g., include suggestions for what to do to avoid the problem, like either include your credentials or log in first)